tests/monster_test_generated.rs

```diff
@@ -176,78 +177,45 @@ pub mod example {
   extern crate flatbuffers;
   use self::flatbuffers::EndianScalar;
 
+bitflags::bitflags! {
 /// Composite components of Monster color.
-#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct Color(pub u8);
-#[allow(non_upper_case_globals)]
-impl Color {
-  pub const ENUM_MIN: u8 = 1;
-  pub const ENUM_MAX: u8 = 8;
-  pub const Red: Self = Self(1);
-  /// \brief color Green
-  /// Green is bit_flag with value (1u << 1)
-  pub const Green: Self = Self(2);
-  /// \brief color Blue (1u << 3)
-  pub const Blue: Self = Self(8);
-  pub const ENUM_VALUES: &'static [Self] = &[
-    Self::Red,
-    Self::Green,
-    Self::Blue,
-  ];
-  /// Returns the variant's name or "" if unknown.
-  pub fn variant_name(self) -> &'static str {
-    match self {
-      Self::Red => "Red",
-      Self::Green => "Green",
-      Self::Blue => "Blue",
-      _ => "",
-    }
-  }
-}
-impl std::fmt::Debug for Color {
-  fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-    let name = self.variant_name();
-    if name.is_empty() {
-      f.write_fmt(format_args!("<UNKNOWN {:?}>", self.0))
-    } else {
-      f.write_str(name)
-    }
+  pub struct Color: u8 {
+    const RED = 1;
+    const GREEN = 2;
+    const BLUE = 8;
   }
 }
+
 impl<'a> flatbuffers::Follow<'a> for Color {
   type Inner = Self;
   #[inline]
   fn follow(buf: &'a [u8], loc: usize) -> Self::Inner {
-    Self(flatbuffers::read_scalar_at::<u8>(buf, loc))
+    let bits = flatbuffers::read_scalar_at::<u8>(buf, loc);
+    unsafe { Self::from_bits_unchecked(bits) }
```

Comment:
This really needs a validator.

Reply:
One more unsound `follow`. Why not.

Reply:
Why don't use [truncating constructor](https://docs.rs/bitflags/1.2.1/bitflags/example_generated/struct.Flags.html#method.from_bits_truncate)? It is safe and only involves one bitwise `and`.

Reply:
Let's resume this conversation after bitflags/bitflags#228